panther_driver/src/driver_node.py

```diff
@@ -117,13 +122,15 @@ def driverNode():
     kinematics_type = rospy.get_param('~wheel_type', "classic")
     odom_frame = rospy.get_param('~odom_frame', "odom")
     base_link_frame = rospy.get_param('~base_link_frame', "base_link")
+    motor_torque_constant = rospy.get_param('~motor_torque_constant', 2.6149)
```

Comment:
If there is a file with parameters (I couldn't find it) please add `motor_torque_constant` there.

Reply:
I see that it is necessary to do it for all parameters, ok to do it later